content/_changelogs/10.8.0.md

```diff
@@ -0,0 +1,106 @@
+  `false`. Fixes [#2939](https://github.com/cypress-io/cypress/issues/2939).
+- Fixed an issue where the stack traces in `run` mode did not match the stack
+  traces shown in `open` mode. Now, In the terminal, users will see the source
+  map line and column number in the error stack trace. Fixes
+  [#7715](https://github.com/cypress-io/cypress/issues/7715).
+- Fix a bug where projects using Node.js 16.17+ and 18.6+ with ES Modules and
+  TypeScript were not working with Cypress. Fixes
+  [#22795](https://github.com/cypress-io/cypress/issues/22795),
+  [#23393](https://github.com/cypress-io/cypress/issues/23393) and
+  [#23552](https://github.com/cypress-io/cypress/issues/23552).
+- When users on Windows use the spec list search, if their search term contains
+  any instances of `/`, we now replaced with `\` to correctly match the Windows
+  path. Fixes [#23426](https://github.com/cypress-io/cypress/issues/23426).
+- Fixed an regression introduced in Cypress [10.3.0](#10-3-0), and further
+  exposed in [10.4.0](#10-4-0), that omitted `same-site` cookies when the URL
+  Scheme, Domain, and Top Level Domain matched, but the ports are different
+  (i,.e. same-site). Fixes
+  [#23132](https://github.com/cypress-io/cypress/issues/23132).
+- Autoscrolling of the command log can no longer become disabled in `run` mode.
+  This fixes issue observed where the reporter randomly stops scrolling during a
+  video recording and the Command log details are not visible. Fixes
+  [#16098](https://github.com/cypress-io/cypress/issues/16098) and
+  [#22199](https://github.com/cypress-io/cypress/issues/22199).
+- The `cypress/react18` [`cy.mount()`](/guides/component-testing/mounting-react)
+  command's yielded rerender function has been updated to only create one
+  `ReactDom.Root` element. This fixes the issue where state did not persist
+  between rerenders, and all lifecycle hooks were reran when executed. Fixes
+  [#23357](https://github.com/cypress-io/cypress/issues/23357).
+- Fixed a regression introduced in Cypress [10.0.0](#10-0-0) where Cypress was
+  incompatible with Chrome v64 - v70. Fixes
+  [#23509](https://github.com/cypress-io/cypress/issues/23644).
+- Fixes an issue where outdated organization and project information could be
+  shown in dialogs when connecting a project to the Cypress Dashboard. Fixes
+  [#23538](https://github.com/cypress-io/cypress/issues/23538)
+- Changes were made to set the `Sec-Fetch-Dest`
+  [metadata request header](https://developer.mozilla.org/en-US/docs/Glossary/Fetch_metadata_request_header)
+  to `document` when the request was sent from the Application Under Test when
+  the
+  [`experimentalModifyObstructiveThirdPartyCode`](/guides/references/experiments#Configuration)
+  experiment is enabled. This was updated to prevent Google Authentication from
+  returning `403 forbidden` in Edge and Electron. Fixes
+  [#23720](https://github.com/cypress-io/cypress/pull/23720).
```

Comment:
```suggestion
- The `Sec-Fetch-Dest`
  [metadata request header](https://developer.mozilla.org/en-US/docs/Glossary/Fetch_metadata_request_header)
  is now set to `document` when the request is sent from the Application Under Test when
  the
  [`experimentalModifyObstructiveThirdPartyCode`](/guides/references/experiments#Configuration)
  experiment is enabled. This was updated to prevent Google Authentication from
  returning `403 forbidden` in Edge and Electron. Fixes
  [#23720](https://github.com/cypress-io/cypress/pull/23720).
```